yumi_description/urdf/yumi.urdf

```diff
@@ -598,13 +520,9 @@
       <material name="Grey"/>
     </collision>
   </link>
-  <!--Extensions -->
-  <!-- body -->
-  <gazebo reference="yumi_body">
```

Comment:
yeah the material stuff is used in `rviz` and the `<gazebo ...>` stuff is used in gazebo. (you have to have both ;))